src/actions/handleGithubComment.js

```diff
@@ -0,0 +1,50 @@
+
+const axios = require('axios');
+const config = require('../../.config')
+module.exports = ({task, conn}) => async function handleGithubComment(params) {
+    const {Subscriber} = conn.models;
+
+    const {comment, issue} = params;
+    const orgs = await axios.get(comment.user['organizations_url']).then(res => res.data);
+    const orgNames = orgs.map(org => org.login);
+    const orgIds = orgs.map(org => org.id);
+    const subscriber = await Subscriber.findOne({
+      $or: [
+        { githubUsername: comment.user.login },
+        { githubUserId: comment.user.id.$numberInt },
+        { githubOrganization: { $in: orgNames } },
+        { githubOrganizationId: { $in: orgIds } },
+        { 'githubOrganizationMembers.login': comment.user.login },
+        { 'githubOrganizationMembers.id': comment.user.id.$numberInt }
+      ]
+    });
+
+    if (subscriber == null) {
+      return { ok: 1 };
+    }
+    await task.log('subscriber commented on issue');
+
+    // Send to Slack
+    const url = 'https://slack.com/api/chat.postMessage';
```

Comment:
Instead of duplicating the Slack logic here, let's refactor the integration from #10 to take an array of `blocks` instead of just the `email`.

`src/actions/requestInvite.js` -> `src/integrations/slack.js`

```javascript
module.exports = async function sendMessage(blocks) {
    const url = 'https://slack.com/api/chat.postMessage';
    await axios.post(url, {
      channel: '#pro-notifications',
      blocks
    }, { headers: { authorization: `Bearer ${config.slackToken}` } });
}
```

That will make it easier to send Slack messages in the future